wrapanapi/systems/rhevm.py

```diff
@@ -860,6 +860,31 @@ def list_cluster(self, **kwargs):
         cluster_list = self.api.system_service().clusters_service().list(**kwargs)
         return [cluster.name for cluster in cluster_list]
 
+    def get_disks(self, name):
+        return self.api.system_service().disks_service().list(search=name)
+
+    def list_disks(self, status=None, **kwargs):
+        """
+        List all of the disks present on RHV
+
+        Keywords:
+            status (optional) -- status of the disk.One of OK, LOCKED, ILLEGAL.
+
+        Returns:
+            list of disk names(str)
+        """
+        disks_list = self.api.system_service().disks_service().list()
+        if not status:
```

Comment:
I think its better to check `if status is None` here, when there is a `None` default.  Its more readable having positive logic, and the identity check against `None` is unique/special.